tests/integrations/celery/test_celery.py

```diff
@@ -316,9 +315,11 @@ def dummy_task(x, y):
 
     assert len(Hub.current._stack) == 1
     if VERSION < (4,):
-        assert stack_lengths == [2]
+        # This new scopes the _stack is deprecated and its length is always 1.
+        assert stack_lengths == [1]
     else:
-        assert stack_lengths == [2, 2]
+        # This new scopes the _stack is deprecated and its length is always 1.
+        assert stack_lengths == [1, 1]
```

Comment:
If we're ultimately merging this into `sentry-sdk-2.0` there's no Celery 3 support there (because not supported for py3.6+) so we could get rid of some of this.

Eventually we'll probably want to get rid of the whole test since it tests an outdated concept (hub stack) or rewrite it to something equivalent in the new scopes world.

Reply:
getting rid of the test is a good idea. removing it.